packages/next/taskfile.js

```diff
@@ -4,33 +4,13 @@ const relative = require('path').relative
 const babelClientOpts = {
   presets: [
     '@babel/preset-typescript',
-    [
-      '@babel/preset-env',
-      {
-        modules: 'commonjs',
-        targets: {
-          esmodules: true,
-        },
-        loose: true,
-        exclude: ['transform-typeof-symbol'],
-      },
-    ],
-    '@babel/preset-react',
+    ['@babel/preset-react', { loose: true, useBuiltIns: true }],
```

Comment:
How about `useSpread` instead of `useBuiltIns`?

Reply:
Good call. Babel merged this into the JSX transform, and [useSpread](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#usespread) is the name they ended up going with.

Reply:
At this point in time `useBuiltIns` is better as it results in `Object.assign ` whereas spread ends up being a helper, currently investigating if we can change that to also compile to Object.assign in loose mode.